Sources/KeyboardShortcuts/Recorder.swift

```diff
@@ -151,18 +174,18 @@ extension KeyboardShortcuts.Recorder {
 
 @available(macOS 10.15, *)
 #Preview {
-	KeyboardShortcuts.Recorder(for: .init("xcodePreview"))
+	KeyboardShortcuts.Recorder("record_shortcut", name: .init("xcodePreview"))
```

Comment:
I used the key of an existing string so the preview wouldn't require a new string to be localized.